src/test/java/Pages/BasePage.java

```diff
@@ -0,0 +1,22 @@
+package Pages;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import java.time.Duration;
+
+
+public class BasePage {
+
+    WebDriver driver;
+    WebDriverWait wait;
+    Actions actions;
```

Comment:
By the way, I recommend: make the variables protected, because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

```
protected  WebDriver driver;
protected WebDriverWait wait;
protected Actions actions;
```